### ISSUE TITLE
/tmp is hardwired, how could I make it work under /dev/shm ?

### ISSUE THREAD

**Issue by fgeorgatos** [qtop/qtop#254]
example:

```
$ grep savepath: qtopconf.yaml
##savepath: $TMPDIR/qtop_results_$USER
savepath: /dev/shm/qtop_results_$USER
$ ./qtop.py -FGTw
Traceback (most recent call last):
  File "./qtop.py", line 2134, in <module>
    scheduler_output_filenames = fetch_scheduler_files(options, config)
  File "./qtop.py", line 566, in fetch_scheduler_files
    scheduler_output_filenames = get_input_filenames(INPUT_FNs_commands)
  File "./qtop.py", line 351, in get_input_filenames
    filenames[_file] = execute_shell_batch_commands(batch_system_commands, filenames, _file)
  File "./qtop.py", line 273, in execute_shell_batch_commands
    with tempfile.NamedTemporaryFile('w', dir=savepath, delete=False) as fin:
  File "/usr/lib64/python2.6/tempfile.py", line 444, in NamedTemporaryFile
    (fd, name) = _mkstemp_inner(dir, prefix, suffix, flags)
  File "/usr/lib64/python2.6/tempfile.py", line 228, in _mkstemp_inner
    fd = _os.open(file, flags, 0600)
OSError: [Errno 2] No such file or directory: '/tmp/qtop_results_fgeorgatos/tmpZdUoqa'
```

**Comment by sfranky** [comment]
fixed in #255

**Comment by fgeorgatos** [comment]
working with thanks!
